docs/dev/cli/saucectl/run.md

```diff
@@ -281,13 +281,66 @@ saucectl run --dry-run
 
 <div className="cli-desc">
 <p><small>| OPTIONAL | KEY=VALUE |</small></p>
-<p><small>| OPTIONAL | BOOLEAN |</small></p>
 An environment variable key value pair that may be referenced in the tests executed by this command. Expanded environment variables are supported.
 
 ```bash
 saucectl run --env <key1>=<value1> --env <key2>=<value2> ...
 ```
 
+#### Supported Metrics
+
+<table id="env">
+  <thead>
+    <tr>
+      <th width="40%">Frameworks and Products</th>
+      <th>Supported?</th>
+      <th>Note</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Cypress</td>
+      <td>✅</td>
+      <td><span className="sauceGreen">Virtual Devices Only</span></td>
+    </tr>
+    <tr>
+      <td>Playwright</td>
+      <td>✅</td>
+      <td><span className="sauceGreen">Virtual Devices Only</span></td>
+    </tr>
+     <tr>
+      <td>TestCafe</td>
+      <td>✅</td>
+      <td><span className="sauceGreen">Virtual Devices Only</span></td>
+    </tr>
+     <tr>
+      <td>Puppeteer Replay</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Espresso</td>
+      <td></td>
+      <td></td>
+    </tr>
```

Comment:
Maybe we should put a ❌ here, to be explicit. Otherwise it may be unclear if it's a mistake ("does empty mean no support or did someone forgot to put an indicator here?").

Reply:
👍 . Let me add it.